src/components/LoadingPage/LoadingPage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const LoadingPage = () => (
+  <div>
+    <img src="/images/loader.gif" alt="spinning circle" data-test="loader-gif" />
```

Comment:
What is the target of this component? When something is loading, there will be redirection to this page, and after loading finished, redirection to the previous one ? 

I think, that we still have some missing point. 

Imagine, that you have a DashboardPage, router is rendering this component. It can have something like this: 
```
<div>
   {isLoading ? <Loader /> : <DashboardContent />}
</div>
```
Ofc, you have to have logic which will start fetching, when DashboardPages is mounted, then setting isLoading to true, and when fetching is done, set `isLoading` to false.